draft-ietf-rats-architecture.md

```diff
@@ -372,13 +261,105 @@ The FIDO protocol supports several remote attestation protocols and a mechanism
 
 Other biometric authentication protocols such as the Chinese IFAA standard and WeChat Pay as well as Google Pay make use of attestation in one form or another.
 
+In this case, every FIDO Authenticator must produce attestation for consideration by any web site, mobile application back end or service that does biometric authentication.
+  
+### Use case terminology
+In the above use cases, the entity producing attestation is called the "Attester" and the entity considering the attestation is called the "Relying Party". A more formal terminology is provided in {{terminology}}.
+
+# Terminology {#terminology}
+
+## Roles
+
 Attester:
 
-: Every FIDO Authenticator contains an Attester.
+: A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation.
+Produces: Evidence.
```

Comment:
I do kind of like the Produces: and Consumes: annotation.  I'd be fine with adding such into the terminology section if others agree.

This PR does make it hard to see changes to the terminology section though, I'd prefer to see a separate PR for such things.